src/main/java/com/ostapula/maven/sort/Person.java

```diff
@@ -35,11 +35,15 @@ public boolean equals(Object o) {
 
 	@Override
 	public int compareTo(Person p) {
+		if (this.name == null && p.getName() == null)
+			return this.surname.replaceAll(" ", "").compareToIgnoreCase(p.getSurname().replaceAll(" ", ""));
```

Comment:
What if both `this.name` and `this.surname` are `null`?

Reply:
https://github.com/Ostapula/Internship/pull/10/commits/a1e9caa2f015e8c6002bd4532844f0d457475c24 in this commit I fixed it.